rubicon/objc/objc.py

```diff
@@ -20,6 +20,7 @@
 ######################################################################
 
 objc = cdll.LoadLibrary(util.find_library('objc'))
+Foundation = cdll.LoadLibrary(util.find_library('Foundation'))
```

Comment:
Leaving out Foundation was a strategic choice; I was trying to avoid the core of Rubicon spiralling into a a huge project. If it's necessary, then we will include it, but I'd like to explore if we can do this with CFArray (from CoreFoundation)